.github/ISSUE_TEMPLATE/bug_report.md

```
@@ -0,0 +1,48 @@
+## Ambiente
+<!-- Preencha os dados relevantes ao seu caso. -->
+- **Sistema operacional:** <!-- ex: Windows 11, macOS 14, Ubuntu 22.04 -->
+- **Navegador / versão:** <!-- ex: Chrome 120, Firefox 121 -->
+- **Ambiente:** <!-- prod/ local -->
+
+
+## Evidências
+<!-- Prints, vídeos, payloads, logs, stack traces ou qualquer material que ajude. -->
+
+
+##  Sugestão de correção (opcional)
```

markdownlint-cli2 (0.23.1)

[warning] Multiple spaces after hash on atx style heading

(MD019, no-multiple-space-atx)